src/components/plots/linePlot.ts

```diff
@@ -70,11 +68,12 @@ export module Plots {
           data.length > 0 ? projector(data[0], i, dataset, m) : null;
       });
 
-      var xFunction = attrToProjector["x"];
-      var yFunction = attrToProjector["y"];
+      var xFunction       = attrToProjector["x"];
+      var yFunction       = attrToProjector["y"];
```

Comment:
tslint doesn't disallow for multiple whitespace.

Also,  how did I make this change?  Definite revert back to old style